exercises/sgf-parsing/canonical-data.json

```diff
@@ -0,0 +1,161 @@
+{
+    "exercise": "sgf-parsing",
+    "version": "1.0.0",
+    "cases": [
+        {
+            "description": "empty input",
+            "property": "parse",
+            "input": {
+                "encoded": ""
+            },
+            "expected": {
+                "error": "tree missing"
+            }
+        },
+        {
+            "description": "tree with no nodes",
+            "property": "parse",
+            "input": {
+                "encoded": "()"
+            },
+            "expected": {
+                "error": "tree no nodes"
+            }
+        },
+        {
+            "description": "node without tree",
+            "property": "parse",
+            "input": {
+                "encoded": ";"
+            },
+            "expected": {
+                "error": "tree missing"
+            }
+        },
+        {
+            "description": "node without properties",
+            "property": "parse",
+            "input": {
+                "encoded": "(;)"
+            },
+            "expected": {
+                "properties": {}
+            }
+        },
+        {
+            "description": "single node tree",
+            "property": "parse",
+            "input": {
+                "encoded": "(;A[B])"
+            },
+            "expected": {
+                "properties": {
+                    "A": ["B"]
+                },
+                "children": []
+            }
+        },
+        {
+            "description": "properties without delimiter",
+            "property": "parse",
+            "input": {
+                "encoded": "(;a)"
```

Comment:
Should the `a` be an uppercase, as the next test checks for lowercase properties?